### ISSUE TITLE
Image not displayed when clicking thumbnail preview

### ISSUE THREAD

**Issue by bkraul** [RocketChat/Rocket.Chat#19563]
### Description:

When an image is attached to a message, clicking on the image thumbnail preview does not display the image. The console returns the following error:

```
d5ab7ee91991237ebf01e25454506aa351ab7b0d.js?meteor_js_resource=true:424 Uncaught (in promise) TypeError: a is not a constructor
    at n (d5ab7ee91991237ebf01e25454506aa351ab7b0d.js?meteor_js_resource=true:424)
```
### Steps to reproduce:

1. Update to latest develop image (3.9.0) (2ab1a10)
2. Attempt to attach an image to a message.
3. Once the image uploads, try clicking on it to view it.

### Expected behavior:

Larger image should be displayed.

### Actual behavior:

Nothing happens, and dev-tools console shows error described above.

**Screen Capture:**
![6vr3JblrDb](https://user-images.githubusercontent.com/14616851/99298646-a7d05180-280f-11eb-8269-22291caa53d8.gif)

### Server Setup Information:

- Version of Rocket.Chat Server: 3.9.0-develop (2ab1a10)
- Operating System: Ubuntu 20.04.1 LTS (docker)
- Deployment Method: docker 
- Number of Running Instances: 1
- DB Replicaset Oplog: Yes
- NodeJS Version: v12.18.4
- MongoDB Version: 4.0.21

### Client Setup Information

- Desktop App or Browser Version: 3.0.7 Desktop (Electron)
- Operating System: Windows 10 Pro 20H2

### Additional context

No additional context needed.

### Relevant logs:

Log from local dev-tools.
```
d5ab7ee91991237ebf01e25454506aa351ab7b0d.js?meteor_js_resource=true:424 Uncaught (in promise) TypeError: a is not a constructor
    at n (d5ab7ee91991237ebf01e25454506aa351ab7b0d.js?meteor_js_resource=true:424)
```

**Comment by bkraul** [comment]
Commit 48af214 does not seem to have this problem.

**Comment by bkraul** [comment]
This appears corrected in subsequent develop tag 0332729 .
